lib/WebAssemblyParser.js

```diff
@@ -4,28 +4,40 @@
 */
 "use strict";
 
-const Tools = require("webassembly-interpreter/lib/tools");
-
-const Tapable = require("tapable").Tapable;
+const {
+	traverse
+} = require("@webassemblyjs/ast");
+const {
+	decode
+} = require("@webassemblyjs/wasm-parser");
+
+const {
+	Tapable
+} = require("tapable");
 const WebAssemblyImportDependency = require("./dependencies/WebAssemblyImportDependency");
 
+const decoderOpts = {
+	ignoreCodeSection: true,
```

Comment:
is there a case when we might want to provide override for these from config?

Reply:
In short term probably not, we can easily expose it after since it's only in Webpack's internals.